core/src/test/java/tech/tablesaw/TableAssertions.java

```diff
@@ -0,0 +1,42 @@
+package tech.tablesaw;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import tech.tablesaw.api.Row;
+import tech.tablesaw.api.Table;
+
+public final class TableAssertions {
+
+  private TableAssertions() {}
+
+  public static void assertTableContents(Object[][] expected, Table actual) {
```

Comment:
It looks like we have a method `assertTablesEquals` in `SortTest` that could be moved to this class

Reply:
I suggest that would be best tracked on a new issue as it's a separate refactoring

Reply:
Took care of this here #743